pkgs/by-name/ca/cargo-clean-recursive/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  cargo,
+  makeWrapper,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-clean-recursive";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "IgaguriMK";
+    repo = pname;
+    rev = "v${version}";
```

```suggestion
    tag = "v${version}";
```

tag attribute directly fetches the tag, whereas rev can fetch a branch with the same name if the repo has any.